install/kubernetes/test.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 
 set -e
+set -o pipefail
 shopt -s expand_aliases
 
+# make this script executable from anywhere
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd $SCRIPT_DIR
+
 alias helm='docker run --rm -v $(pwd):/apps alpine/helm:3.3.4'
+alias kubeval='docker run --rm -i garethr/kubeval:0.15.0'
```

Comment:
I'm always wary of using unofficial docker images.
Could we use binaries from https://github.com/instrumenta/kubeval/releases instead?

Reply:
Ah! I got it from https://www.kubeval.com/installation/, I thought it was "the official".

Reply:
this project is not maintained by the way :/. It was mostly so that I can run in the PostUpgradeCommand without having to install stuff without Docker but we can also install stuff.

Reply:
Since it's not super clear if it's official or not I should pin the sha256 so that everything stays in place for sure.

Reply:
see https://github.com/cilium/tetragon/pull/1036/files#diff-04ccead65ce7bcd231732b9157e93b204f96488a6812f347c9cdc5f070cb028bR12